controller/src/main/java/io/pravega/controller/server/security/auth/StrongPasswordProcessor.java

```diff
@@ -7,7 +7,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.pravega.controller.server.rpc.auth;
+package io.pravega.controller.server.security.auth;
```

Comment:
Should this class be in controller or be moved to "auth" module. 
This has nothing specific to controller's auth.

Reply:
I've created a separate issue #5118 that should include these changes plus other organizational changes required in the security-related code.